Source/WebCore/rendering/svg/SVGTextMetrics.cpp

```diff
@@ -54,7 +54,6 @@ SVGTextMetrics::SVGTextMetrics(RenderSVGInlineText& textRenderer, const TextRun&
     m_glyph.unicodeString = run.text().toString();
     m_glyph.isValid = true;
 
-    ASSERT(run.length() >= 0);
```

Comment:
This is already taken care of in https://github.com/WebKit/WebKit/pull/26684

Reply:
Sorry @Serial-ATA , I was not aware of your MR :-/